helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# Per environment values which override defaults in calculate-release-dates-api/values.yaml
+
+generic-service:
+  replicaCount: 2
+
+  ingress:
+    host: calculate-release-dates-api-preprod.hmpps.service.justice.gov.uk
+
+  env:
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
+    HMPPS_AUTH_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
+    PRISON_API_URL: https://api-preprod.prison.service.justice.gov.uk
+
+# CloudPlatform AlertManager receiver to route promethues alerts to slack
+# See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
+generic-prometheus-alerts:
+  alertSeverity: digital-prison-service-dev
```

Comment:
We need [our own AlertManager](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#create-a-slack-webhook-and-amend-alertmanager) at some point so all of the Kube failure alerts go to #farsight_alerts. We've got a ticket on our backlog but not prioritised yet!

Reply:
Thanks @mikehalmamoj - will add a ticket to our backlog for this